### PR TITLE
[Compat]: Create: Garnished Additions Compatibility (Another addon to Garnished.)

### PR DESCRIPTION
**Pull request by Apocip0**
- Three materials added. That's it.
- Optional Dependency added.

METAL:
- Nutium Ingot

SOFT_PLANT:
- Ethereal Roots

WOOD:
- Ethereal

### CODE REVIEW

**Review by SigmundGranaas — CHANGES_REQUESTED** [review]
For some reason I can not get this to activate when I have garnished additions installed. Can someone double check?

File: fabric/build.gradle [hunk 1]
```diff
@@ -165,6 +166,7 @@ modrinth {
 		optional.project "ecologics"
 		optional.project "naturalist"
 		optional.project "advanced-netherite"
+		optional.project "garnished_additions"
```
Comment (SigmundGranaas) [inline fabric/build.gradle]: Correct one is: `garnished-additions`

File: fabric/build.gradle [hunk 2]
```diff
@@ -126,6 +126,7 @@ curseforge {
 				optionalDependency "ecologics"
 				optionalDependency "naturalist"
 				optionalDependency "advanced-netherite"
+				optionalDependency "garnished_additions"
```
Comment (SigmundGranaas) [inline fabric/build.gradle]: Correct one is: `garnished-additions`

**Comment by Apocip0** [comment]
> For some reason I can not get this to activate when I have garnished additions installed. Can someone double check?

I double-checked and everything seems to work fine, I don't know what's your exact issue. Perhaps you need to add Create itself for it to work if you haven't already since it's a required dependency. It's meant to work together.

**Pull request merged**